src/Core/src/Handlers/View/ViewHandlerOfT.cs

```diff
@@ -95,47 +89,36 @@ public virtual void SetVirtualView(IView view)
 			_mapper.UpdateProperties(this, VirtualView);
 		}
 
-		protected virtual void ConnectHandler(TNativeView nativeView)
+		void IViewHandler.DisconnectHandler()
 		{
-
+			if (NativeView != null && VirtualView != null)
+				DisconnectHandler(NativeView);
 		}
 
-		protected virtual void DisconnectHandler(TNativeView nativeView)
+		protected virtual void ConnectHandler(TNativeView nativeView)
 		{
-
+			base.ConnectHandler(nativeView);
 		}
 
-		void IViewHandler.DisconnectHandler()
+		protected virtual void DisconnectHandler(TNativeView nativeView)
 		{
-			if (TypedNativeView != null && VirtualView != null)
-				DisconnectHandler(TypedNativeView);
-
-			if (VirtualView != null)
-				VirtualView.Handler = null;
-
-			VirtualView = null;
+			base.DisconnectHandler(nativeView);
 		}
```

Comment:
Should this be override? Is this needed?

Reply:
The type on this are different than the base class version of these. The ViewHandler ones are just typed to UIView/View/FrameworkElement

Basically I'm just doing a lazy person's covariance here. I'm just trying to make a covariant version of DisconnectHandler/ConnectHandler so that implementing classes from 3rd party libraries will only see the Typed ones